tests/test_codelists.py

```diff
@@ -58,6 +58,9 @@ class Cohort(OldCohortWithPopulation):
 
 @pytest.mark.integration
 def test_codelist_equals_query(engine):
+    if engine.name == "spark":
+        pytest.xfail()
```

Comment:
There are _juuuuust_ enough of these for me to wonder if it could be factored out into a decorator.

Reply:
I'm hoping they'll be short-lived enough for it not to be worth it...